cea/datamanagement/district_geometry_helper.py

```diff
@@ -0,0 +1,238 @@
+"""
```

Comment:
Why don't we call this tool `create-district-geometry`? I think that might communicate the intent a bit better. What do you think? The same for the terrain-helper and the other one...

Reply:
in principle yes, however we have the data-helper already. maybe we change it all later?

Reply:
sure. i like consistency. i couldn't find the issue i once created for the data-helper to be renamed. we can do that someday when we have time...